x-pack/plugins/lens/public/state_management/index.ts

```diff
@@ -40,7 +40,7 @@ export const {
 
 export const makeConfigureStore = (
   storeDeps: LensStoreDeps,
-  preloadedState: DeepPartial<LensState>
+  preloadedState: PreloadedState<LensState>
```

Comment:
Note to reviewers: This change is required because of the upgrade of `@reduxjs/toolkit` to the v1.6.x line. See https://github.com/reduxjs/redux-toolkit/pull/949 for details.